sinks/lightstep/README.md

```diff
@@ -0,0 +1,32 @@
+# LightStep Sink
+
+This sink sends Veneur spans to [LightStep](https://lightstep.com).
```

Comment:
One other thing we should probably write here is what capabilities each sink implements / what purpose you can use it for, just like this one (maybe in some common format? list?)